tests/integration/models/reddit/test_submission.py

```diff
@@ -328,6 +328,17 @@ def test_flair(self, _):
         with self.recorder.use_cassette("TestSubmissionModeration.test_flair"):
             self.reddit.submission("4b536p").mod.flair("submission flair")
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_flair_template_id(self, _):
```

Comment:
I'd like to see some unit tests that test various configurations of arguments given to the function and whether they are valid or not.

Reply:
Ahh. Ok.